src/engine.jl

```diff
@@ -40,53 +36,49 @@ type MSession
             # Hide the MATLAB Command Window on Windows
             ccall(engfunc(:engSetVisible ), Cint, (Ptr{Void}, Cint), ep, 0)
         end
-
-        println("A MATLAB session is open successfully")
         new(ep, buf, bufptr)
     end
-
-    MSession() = MSession(default_output_buffer_size)
 end
 
+
 function close(session::MSession)
     # Close a MATLAB Engine session
-
     @assert libeng::Ptr{Void} != C_NULL
-
     r = ccall(engfunc(:engClose), Cint, (Ptr{Void},), session.ptr)
```

Comment:
I'm not 100% sure why it's better to null it before the pointer to the call.

Anyways,  please see the latest push (first commit  on the branch).

After this, I have addressed all of your review comments.

Thanks for the review.

In the future, I may think of registering something like close as a  finalizer for MSession?